src/Domain/Creators/Sphere.cpp

```diff
@@ -267,11 +267,20 @@ Sphere::Sphere(
       // Build the maps. We only apply the maps in the inner-most shell. The
       // inner radius is what's passed in, but the outer radius is the outer
       // radius of the inner-most shell so we have to see how many shells we
-      // have
+      // have.
+      // The translation map linear transition occurs only in the outer-most
+      // shell, such that the outer boundary is not translated. If there
+      // is no radial partition, a uniform translation is applied instead, as
+      // the transition requires spherical shape.
```

Comment:
I don't think it makes sense to allow only a single spherical shell while also having a translation. I don't think we have boundary conditions that can handle that. We should probably just error if that happens and require we have two spherical shells (one radial partition).